core/src/components/button/test/shape/button.e2e.ts

```diff
@@ -38,42 +113,54 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
 
           await page.setIonViewport();
 
-          const container = page.locator('#clear');
+          const container = page.locator('#round #clear');
 
-          await expect(container).toHaveScreenshot(screenshot(`button-clear-round`));
+          await expect(container).toHaveScreenshot(screenshot(`button-round-clear`));
         });
       });
+    });
 
-      test.describe('color', () => {
+    test.describe('rectangular', () => {
+      test.describe('default', () => {
         test('should not have visual regressions', async ({ page }) => {
           await page.goto(`/src/components/button/test/shape`, config);
 
           await page.setIonViewport();
 
-          const container = page.locator('#color');
+          const container = page.locator('#rectangular #default');
 
-          await expect(container).toHaveScreenshot(screenshot(`button-color-round`));
+          await expect(container).toHaveScreenshot(screenshot(`button-rectangular`));
         });
       });
 
-      test.describe('expand', () => {
+      test.describe('outline', () => {
         test('should not have visual regressions', async ({ page }) => {
           await page.goto(`/src/components/button/test/shape`, config);
 
           await page.setIonViewport();
 
-          const container = page.locator('#expand');
+          const container = page.locator('#rectangular #outline');
 
-          await expect(container).toHaveScreenshot(screenshot(`button-expand-round`));
+          await expect(container).toHaveScreenshot(screenshot(`button-rectangular-outline`));
+        });
+      });
+
+      test.describe('clear', () => {
+        test('should not have visual regressions', async ({ page }) => {
+          await page.goto(`/src/components/button/test/shape`, config);
+
+          await page.setIonViewport();
+
+          const container = page.locator('#rectangular #clear');
+
+          await expect(container).toHaveScreenshot(screenshot(`button-rectangular-clear`));
         });
       });
     });
   });
 });
 
-/**
- * Shape="rectangular" is only available in the Ionic theme.
- */
+// TODO do we need this test for focused / activated? it should be for all themes if so
```

Comment:
Need to determine if we need this test or if there is a way to slim it down / move it.

Reply:
I ended up removing this test because the activated state does not get changed based on the shape & we are checking that the focused state is the proper shape in the clear button tests.